airflow/utils/helpers.py

```diff
@@ -182,20 +182,25 @@ def f(t):
     return s
 
 
-def kill_using_shell(pid, signal=signal.SIGTERM):
+def kill_using_shell(logger, pid, signal=signal.SIGTERM):
     process = psutil.Process(pid)
     # Use sudo only when necessary - consider SubDagOperator and SequentialExecutor case.
     if process.username() != getpass.getuser():
         args = ["sudo", "kill", "-{}".format(int(signal)), str(pid)]
     else:
         args = ["kill", "-{}".format(int(signal)), str(pid)]
     # PID may not exist and return a non-zero error code
-    subprocess.call(args)
+    try:
+        subprocess.check_output(args)
+        logger.info("Killed process {} with signal {}".format(pid, signal))
```

Comment:
It shoudl remain in the try except so it doesn't get executed in the exceptional case